### PR TITLE
Usage of pycodestyle instead of pep8

### PR DESCRIPTION
**Pull request by Trybnetic**
This Pull Request updates the `tox.ini` configuration file.

It changes the style checker from pep8 to [pycodestyle](https://pycodestyle.readthedocs.io/en/latest/) since pep8 is depreciated.

### CODE REVIEW

**Comment by dekuenstle** [comment]
There is a warning thrown while the docs part. I think make has explicitly to be imported for tox. 

```
docs runtests: commands[0] | make html
WARNING:test command found but not installed in testenv
  cmd: /bin/make
  env: /home/dek/gitroot/pyndl/.tox/docs
Maybe you forgot to specify a dependency? See also the whitelist_externals envconfig setting.
```

File: tox.ini [hunk 1]
```diff
@@ -5,13 +5,14 @@ envlist = py{34,35,36}-test, checkstyle, docs
 usedevelop = True
 deps =
      test: pytest
+     test: -rrequirements.txt
```
Comment (dekuenstle) [inline tox.ini]: Why did you add this part? 
For me everything works (locally), even if i comment it out. I think, tox uses the package install (setup.py) to install the dependencies via requirements.txt anyway

Comment (Trybnetic) [reply tox.ini]: I added this line because it did not work for me without. @derNarr didn't you have the same problem as well?

However, I am currently not able to reproduce the error anymore

**Comment by Trybnetic** [comment]
@dekuenstle I whitelisted `/usr/bin/make` for docs in `tox.ini` as [you suggested](https://github.com/quantling/pyndl/pull/110#issuecomment-350003153). As mentioned, adding installing the requirements, solved a problem on my machine, although I am not able to reproduce the error anymore.

But you are right, with `usedevelop = True` all dependencies should already be installed before running the dependency install.

File: tox.ini [hunk 2]
```diff
@@ -40,6 +40,7 @@ ignore_outcome = True
 
 [testenv:docs]
 changedir = doc
+whitelist_externals=/usr/bin/make
```
Comment (derNarr) [inline tox.ini]: Should the Makefile contain a make target `doc` or `docs`? At the moment, one only can invoke the build of the documentation with `tox -e docs`.

Comment (Trybnetic) [reply tox.ini]: I agree with you on this issue and I already added it in #112. However, I am not sure, whether I would add this in this PR because currently it is more concerned with maintaining issues as pep8 is depreciated now

**Comment by Trybnetic** [comment]
@dekuenstle I think your comments are resolved, aren't they?
